build.gradle.kts

```diff
@@ -17,8 +17,8 @@ buildscript {
 }
 
 plugins {
-  kotlin("jvm") version "1.4.21" apply false
-  id("org.jetbrains.kotlin.plugin.allopen") version "1.4.21" apply false
+  kotlin("jvm") version "1.4.31" apply false
```

Comment:
@robfletcher Is there a way we could avoid specifying the version here and just pick up the version defined in kork?

Reply:
No

Reply:
K

Reply:
It's annoying but it's not possible to specify Gradle plugin versions with a variable / property because they get loaded before anything else (including project properties). It's a long-standing gripe.